docs/src/modules/components/AppFrame.js

```diff
@@ -64,6 +64,8 @@ export function NextNProgressBar() {
   return <NProgressBar />;
 }
 
+const sx = { minWidth: { sm: 160 } };
```

Comment:
In https://github.com/mui/material-ui/pull/36160, a `minWidth` of `150` was used, was the change to `160` intentional?

Reply:
It was 200 before, Danilo moved it to 150. 160 makes the surface area to click on a bit larger. No real preferences.